misc-tools/import-accounts-multiuser.py

```diff
@@ -0,0 +1,81 @@
+#!/usr/bin/env python3
+
+import sys, csv, os, re, json
+
+api_url = sys.argv[1]
+httpie_options = sys.argv[2]
+
+# Note: this script uses httpie so we can use the same commandline flags as the base import-contest.sh script
+
+# Import all non-team accounts. We use the existing CSV import functionality for this,
+# because for jury members it creates teams and a category, which we can only do in the
+# API if we know a contest ID. We do not know one at this point
+def import_non_team_accounts():
+    with open('accounts.tsv') as csv_file:
+        with open('accounts_non_team.tsv', 'w') as csv_output:
```

Comment:
See tyhe comment above this method for that reason. If you have a better idea it is very welcome.

Reply:
Just double checking: do we actually need the contest ID in the API to create what we do create or can we get it of that restriction instead?

Reply:
The API requires it. There is no underlying reason. But I think it is hard to get rid of it, since we use the api/contests/xxx prefix everywhere. I can investigate if we can get rid of it but I highly doubt it. Especially for teams which can differ per contest